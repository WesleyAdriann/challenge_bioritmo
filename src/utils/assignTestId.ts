type componentType = 'div' | 'text' | 'input' | 'button' | 'img'

export const assignTestId = (
  componentType: componentType,
  componentId: string
)  => ({
    'data-testid': `${componentType}_${componentId}`,
  })


export const mockTestID = (
    componentType: componentType,
    componentId: string
  ) => assignTestId(componentType, componentId)
