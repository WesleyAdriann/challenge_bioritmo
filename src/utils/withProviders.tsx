
import React from 'react'

import { ThemeProvider } from 'styled-components'

import { theme } from '../style'

export const StorybookProviders: React.FC = ({
  children,
}) => (
  <ThemeProvider theme={theme}>
    {children}
  </ThemeProvider>
)
