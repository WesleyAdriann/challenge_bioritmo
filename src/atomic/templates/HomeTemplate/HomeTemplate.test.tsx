import React from 'react'
import { render, cleanup } from '@testing-library/react'

import { mockTestID, StorybookProviders as wrapper } from '../../../utils'

import { HomeTemplate, IHomeTemplate } from './index'

describe('Templates/HomeTemplate', () => {
  const props: IHomeTemplate = {
    testID: 'IHomeTemplate',
    form: {
      onClear: jest.fn(),
      onSubmit: jest.fn()
    },
    isSearch: false,
    locations: []
  }

  const renderComponent = () => render(<HomeTemplate {...props} />, { wrapper })

  let component = renderComponent()

  beforeEach(() => {
    jest.clearAllMocks()
    cleanup()
    component = renderComponent()
  })

  const el_container = mockTestID('div', props.testID!)

  it(`should render component #${el_container}`, async () => {
    const sut = await component.findByTestId(el_container)
    expect(sut).toBeTruthy()
  })

  it('should render snapshot', () => {
    expect(component.container.firstChild).toMatchSnapshot()
  })
})
