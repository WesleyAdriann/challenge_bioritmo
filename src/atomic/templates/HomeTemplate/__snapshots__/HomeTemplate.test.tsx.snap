// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Templates/HomeTemplate should render snapshot 1`] = `
.c4 {
  color: #333333;
  font-weight: bold;
  font-size: 32px;
  text-align: left;
  text-transform: uppercase;
  -webkit-user-select: initial;
  -moz-user-select: initial;
  -ms-user-select: initial;
  user-select: initial;
  display: block;
  line-height: normal;
}

.c6 {
  color: #333333;
  font-weight: 400;
  font-size: 14px;
  text-align: left;
  text-transform: none;
  -webkit-user-select: initial;
  -moz-user-select: initial;
  -ms-user-select: initial;
  user-select: initial;
  display: block;
  line-height: 1.6;
}

.c10 {
  color: #000;
  font-weight: 400;
  font-size: 14px;
  text-align: left;
  text-transform: none;
  -webkit-user-select: initial;
  -moz-user-select: initial;
  -ms-user-select: initial;
  user-select: initial;
  display: block;
  line-height: normal;
}

.c13 {
  color: #000;
  font-weight: 400;
  font-size: 14px;
  text-align: left;
  text-transform: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  display: block;
  line-height: normal;
}

.c20 {
  color: #000;
  font-weight: 400;
  font-size: 14px;
  text-align: center;
  text-transform: none;
  -webkit-user-select: initial;
  -moz-user-select: initial;
  -ms-user-select: initial;
  user-select: initial;
  display: block;
  line-height: normal;
}

.c21 {
  color: #000;
  font-weight: bold;
  font-size: 20px;
  text-align: left;
  text-transform: none;
  -webkit-user-select: initial;
  -moz-user-select: initial;
  -ms-user-select: initial;
  user-select: initial;
  display: inline;
  line-height: normal;
}

.c28 {
  color: #000;
  font-weight: bold;
  font-size: 14px;
  text-align: left;
  text-transform: none;
  -webkit-user-select: initial;
  -moz-user-select: initial;
  -ms-user-select: initial;
  user-select: initial;
  display: block;
  line-height: normal;
}

.c34 {
  color: #fff;
  font-weight: 400;
  font-size: 14px;
  text-align: left;
  text-transform: none;
  -webkit-user-select: initial;
  -moz-user-select: initial;
  -ms-user-select: initial;
  user-select: initial;
  display: block;
  line-height: normal;
}

.c7 {
  background-color: #fff;
  border-radius: 8px;
  box-shadow: 0px;
  border: 4px solid #F3F3F3;
  padding: 16px;
}

.c25 {
  background-color: #F3F3F3;
  border-radius: 0px;
  box-shadow: 0px;
  border: 0px solid #F3F3F3;
  padding: 12px;
}

.c30 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c31 {
  height: 60px;
  margin-bottom: 8px;
}

.c23 {
  background-color: #FFB612;
  border-radius: 4px;
  border-color: #FFB612;
  border-width: 1px;
  border-style: solid;
  cursor: pointer;
  font-weight: bold;
  padding: 12px;
  text-align: center;
  text-transform: uppercase;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  word-wrap: break-word;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c23:active {
  -webkit-filter: brightness(95%);
  filter: brightness(95%);
}

.c24 {
  background-color: #fff;
  border-radius: 4px;
  border-color: #808080;
  border-width: 1px;
  border-style: solid;
  cursor: pointer;
  font-weight: bold;
  padding: 12px;
  text-align: center;
  text-transform: uppercase;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  word-wrap: break-word;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c24:active {
  -webkit-filter: brightness(95%);
  filter: brightness(95%);
}

.c15 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: transparent;
  border: 3px solid #808080;
  border-radius: 50%;
  cursor: pointer;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 20px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 2px;
  width: 20px;
}

.c15:active {
  -webkit-filter: brightness(105%);
  filter: brightness(105%);
}

.c19 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: transparent;
  border: 3px solid #808080;
  border-radius: 4px;
  cursor: pointer;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 20px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 20px;
}

.c19:active {
  -webkit-filter: brightness(105%);
  filter: brightness(105%);
}

.c11 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-bottom: 1px solid #F3F3F3;
  cursor: initial;
  margin-bottom: 16px;
  padding: 8px 0;
}

.c14 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-bottom: 1px solid #F3F3F3;
  cursor: pointer;
  margin-bottom: 0px;
  padding: 8px 0;
}

.c14:active {
  -webkit-filter: brightness(105%);
  filter: brightness(105%);
}

.c16 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-bottom: 1px solid #F3F3F3;
  cursor: pointer;
  margin-bottom: 16px;
  padding: 8px 0;
}

.c16:active {
  -webkit-filter: brightness(105%);
  filter: brightness(105%);
}

.c18 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-bottom: 1px solid transparent;
  cursor: pointer;
  margin-bottom: 0px;
  padding: 8px 0;
}

.c18:active {
  -webkit-filter: brightness(105%);
  filter: brightness(105%);
}

.c12 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 8px;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c8 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 12px;
  padding-bottom: 16px;
}

.c9 {
  height: 34px;
  width: 34px;
}

.c17 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 16px;
}

.c22 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 22px;
  margin: auto;
  margin-top: 16px;
  max-width: 770px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c1 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #000;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c2 {
  margin: auto;
  max-width: 1200px;
  padding: 20px;
  width: 100%;
}

.c3 {
  margin: 16px auto;
}

.c5 {
  background-color: #333333;
  height: 12px;
  margin: 22px 0 38px;
  width: 92px;
}

.c26 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  gap: 24px;
  padding: 24px 0 12px;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c27 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c29 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 16px;
  margin-top: 12px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c32 {
  gap: 20px;
  display: grid;
  grid-template-columns: 1fr;
}

.c33 {
  background-color: #333333;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 60px 0;
}

@media (min-width:700px) {
  .c17 {
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
    -webkit-box-pack: justify;
    -webkit-justify-content: space-between;
    -ms-flex-pack: justify;
    justify-content: space-between;
  }
}

@media (min-width:700px) {
  .c22 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
  }
}

@media (min-width:700px) {
  .c26 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
  }
}

@media (min-width:700px) {
  .c32 {
    grid-template-columns: 1fr 1fr;
  }
}

@media (min-width:1100px) {
  .c32 {
    grid-template-columns: 1fr 1fr 1fr;
  }
}

<div
  class="c0"
  data-testid="div_IHomeTemplate"
>
  <div
    class="c1"
  >
    <svg
      height="10rem"
      id="prefix__Layer_1"
      viewBox="0 0 146 54"
      width="10rem"
      x="0"
      xml:space="preserve"
      y="0"
    >
      <style>
        .prefix__st0{fill-rule:evenodd;clip-rule:evenodd;fill:#fcb823}
      </style>
      <path
        class="prefix__st0"
        d="M95 38.5V10.7c0-9.5 9.9-11.5 17-8.8v5.3c-4.9-2-10.4-1.5-10.4 4.6h7.9v5.3h-7.9v21.3l-6.6.1zM126.5 7.7h4.9l1 4.1h6.1v5.3h-5.4v12.3c0 2.5 1.1 3.7 3.2 3.7h3.4v5.3h-6.6c-4.4 0-6.7-2.4-6.7-7.1V7.7zM114.7 11.8h6.7v26.6h-6.7zM114.7 2.2h6.7v5.5h-6.7z"
      />
      <path
        class="prefix__st0"
        d="M145.4 38.4c-5.9 9.3-16.2 15.4-28 15.4s-22.1-6.1-28-15.4c6.9 6.4 16.9 10.3 28 10.3 11.1.1 21.1-3.9 28-10.3"
      />
      <path
        d="M.8 32.3v-3.6c1.9.8 4.2 1.2 6.7 1.2 1.9 0 2.8-.6 2.8-1.8 0-1.1-.6-1.7-1.9-1.7H5.3c-3.5 0-5.3-1.8-5.3-5.3 0-3.6 2.6-5.4 7.7-5.4 2.2 0 4.2.3 6.2 1v3.6c-1.9-.8-4-1.2-6.3-1.2-2.4 0-3.5.6-3.5 1.8 0 1.1.7 1.7 2.1 1.7H9c3.8 0 5.8 1.8 5.8 5.3 0 3.6-2.5 5.4-7.4 5.4-2.5 0-4.6-.4-6.6-1m16.9.8V15.8h3.4l.5 2.2c2-1.5 4-2.2 5.9-2.2 2.2 0 3.7.8 4.6 2.4 2.2-1.6 4.3-2.4 6.2-2.4 3.7 0 5.5 2.2 5.5 6.6v10.7h-4.4V22.2c0-1.9-.8-2.9-2.4-2.9-1.3 0-2.7.7-4.1 2.2v11.6h-4.4V22.2c0-2-.8-2.9-2.4-2.9-1.4 0-2.8.7-4.1 2.2v11.6h-4.3zm29.4-5.4c0-3.6 2.5-5.3 7.5-5.3 1.5 0 2.9.1 4 .3v-1.1c0-1.5-1.5-2.3-4.5-2.3-1.9 0-3.8.3-5.6.8v-3.5c1.9-.5 3.8-.8 5.6-.8 5.9 0 8.9 1.9 8.9 5.7v11.6h-2.2l-1.9-1.2c-1.5.8-3.1 1.2-4.8 1.2-4.7 0-7-1.8-7-5.4zM58.6 26c-1.1-.2-2.4-.3-4-.3-2.1 0-3.2.7-3.2 2s.9 2 2.6 2c1.8 0 3.3-.5 4.5-1.4V26zm8.7 7.1V15.8h3.4l.5 2.2c1.5-1.5 3.1-2.2 4.8-2.2v3.5c-1.6 0-3.1.7-4.4 2.2v11.6h-4.3zM78.7 13h3.2l.7 2.8h4v3.5h-3.5v8c0 1.6.7 2.4 2.1 2.4h1.4v3.5H83c-2.9 0-4.4-1.6-4.4-4.7V13z"
        fill="#fff"
      />
    </svg>
  </div>
  <section
    class="c2 c3"
  >
    <p
      class="c4"
      color="#333333"
      data-testid="text_Text"
      size="32"
    >
      Reabertura
      <br />
      Smart Fit
    </p>
    <div
      class="c5"
    />
    <p
      class="c6"
      color="#333333"
      data-testid="text_Text"
    >
      O horário de funcionamento das nossas unidades está seguindo os decretos de cada município. Por isso, confira aqui se a sua unidade está aberta e as medidas de segurança que estamos seguindo.
    </p>
  </section>
  <section
    class="c2"
  >
    <div
      class="c7"
      color="#fff"
      data-testid="div_FormSelectHour"
      radius="8"
    >
      <div
        class="c8"
      >
        <img
          alt="clock"
          class="c9"
          src="icon-hour.png"
        />
        <p
          class="c10"
          data-testid="text_Text"
        >
          Horário
        </p>
      </div>
      <div
        class="c11"
        data-testid="div_ListItem"
      >
        <div
          class="c12"
        >
          <p
            class="c13"
            data-testid="text_ListItem_text"
          >
            Qual período quer treinar?
          </p>
        </div>
        <p
          class="c13"
          data-testid="text_ListItem_right_text"
        />
      </div>
      <div
        class="c14 "
        data-testid="div_ListItem"
      >
        <div
          class="c12"
        >
          <div
            class="c15"
            data-testid="div_ListItem_radio"
            size="20"
          />
          <p
            class="c13"
            data-testid="text_ListItem_text"
          >
            Manhã
          </p>
        </div>
        <p
          class="c13"
          data-testid="text_ListItem_right_text"
        >
          06:00 às 12:00
        </p>
      </div>
      <div
        class="c14 "
        data-testid="div_ListItem"
      >
        <div
          class="c12"
        >
          <div
            class="c15"
            data-testid="div_ListItem_radio"
            size="20"
          />
          <p
            class="c13"
            data-testid="text_ListItem_text"
          >
            Tarde
          </p>
        </div>
        <p
          class="c13"
          data-testid="text_ListItem_right_text"
        >
          12:01 às 18:00
        </p>
      </div>
      <div
        class="c16 "
        data-testid="div_ListItem"
      >
        <div
          class="c12"
        >
          <div
            class="c15"
            data-testid="div_ListItem_radio"
            size="20"
          />
          <p
            class="c13"
            data-testid="text_ListItem_text"
          >
            Noite
          </p>
        </div>
        <p
          class="c13"
          data-testid="text_ListItem_right_text"
        >
          18:01 às 23:00
        </p>
      </div>
      <div
        class="c17"
      >
        <div
          class="c18"
          data-testid="div_ListItem"
        >
          <div
            class="c12"
          >
            <div
              class="c19"
              data-testid="div_ListItem_checkbox"
              size="20"
            />
            <p
              class="c13"
              data-testid="text_ListItem_text"
            >
              Exibir unidades fechadas
            </p>
          </div>
          <p
            class="c13"
            data-testid="text_ListItem_right_text"
          />
        </div>
        <p
          class="c20"
          data-testid="text_Text"
        >
          Resultados encontrados:
           
          <span
            class="c21"
            data-testid="text_Text"
            size="20"
          >
            0
          </span>
        </p>
      </div>
      <div
        class="c22"
      >
        <div
          class="c23"
          data-testid="button_Button"
        >
          Encontrar unidade
        </div>
        <div
          class="c24"
          data-testid="button_Button"
        >
          Limpar
        </div>
      </div>
    </div>
  </section>
  <section
    class="c2"
  >
    <div
      class="c25"
      data-testid="div_Card"
    >
      <div
        class="c26"
      >
        <div
          class="c27"
        >
          <p
            class="c28"
            data-testid="text_Text"
          >
            Máscara
          </p>
          <div
            class="c29"
          >
            <div
              class="c30"
              data-testid="div_ImageWithLegend"
            >
              <img
                class="c31"
                src="required-mask.png"
              />
              <p
                class="c10"
                data-testid="text_Text"
              >
                Obrigatório
              </p>
            </div>
            <div
              class="c30"
              data-testid="div_ImageWithLegend"
            >
              <img
                class="c31"
                src="recommended-mask.png"
              />
              <p
                class="c10"
                data-testid="text_Text"
              >
                Recomendado
              </p>
            </div>
          </div>
        </div>
        <div
          class="c27"
        >
          <p
            class="c28"
            data-testid="text_Text"
          >
            Toalha
          </p>
          <div
            class="c29"
          >
            <div
              class="c30"
              data-testid="div_ImageWithLegend"
            >
              <img
                class="c31"
                src="required-towel.png"
              />
              <p
                class="c10"
                data-testid="text_Text"
              >
                Obrigatório
              </p>
            </div>
            <div
              class="c30"
              data-testid="div_ImageWithLegend"
            >
              <img
                class="c31"
                src="recommended-towel.png"
              />
              <p
                class="c10"
                data-testid="text_Text"
              >
                Recomendado
              </p>
            </div>
          </div>
        </div>
        <div
          class="c27"
        >
          <p
            class="c28"
            data-testid="text_Text"
          >
            Bebedouro
          </p>
          <div
            class="c29"
          >
            <div
              class="c30"
              data-testid="div_ImageWithLegend"
            >
              <img
                class="c31"
                src="partial-fountain.png"
              />
              <p
                class="c10"
                data-testid="text_Text"
              >
                Parcial
              </p>
            </div>
            <div
              class="c30"
              data-testid="div_ImageWithLegend"
            >
              <img
                class="c31"
                src="forbidden-fountain.png"
              />
              <p
                class="c10"
                data-testid="text_Text"
              >
                Proibido
              </p>
            </div>
          </div>
        </div>
        <div
          class="c27"
        >
          <p
            class="c28"
            data-testid="text_Text"
          >
            Vestiários
          </p>
          <div
            class="c29"
          >
            <div
              class="c30"
              data-testid="div_ImageWithLegend"
            >
              <img
                class="c31"
                src="required-lockerroom.png"
              />
              <p
                class="c10"
                data-testid="text_Text"
              >
                Liberado
              </p>
            </div>
            <div
              class="c30"
              data-testid="div_ImageWithLegend"
            >
              <img
                class="c31"
                src="partial-lockerroom.png"
              />
              <p
                class="c10"
                data-testid="text_Text"
              >
                Parcial
              </p>
            </div>
            <div
              class="c30"
              data-testid="div_ImageWithLegend"
            >
              <img
                class="c31"
                src="forbidden-lockerroom.png"
              />
              <p
                class="c10"
                data-testid="text_Text"
              >
                Fechado
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    class="c2 c32"
  />
  <div
    class="c1 c33"
  >
    <svg
      height="4rem"
      id="prefix__Layer_1"
      viewBox="0 0 146 54"
      width="8rem"
      x="0"
      xml:space="preserve"
      y="0"
    >
      <style>
        .prefix__st0{fill-rule:evenodd;clip-rule:evenodd;fill:#fcb823}
      </style>
      <path
        class="prefix__st0"
        d="M95 38.5V10.7c0-9.5 9.9-11.5 17-8.8v5.3c-4.9-2-10.4-1.5-10.4 4.6h7.9v5.3h-7.9v21.3l-6.6.1zM126.5 7.7h4.9l1 4.1h6.1v5.3h-5.4v12.3c0 2.5 1.1 3.7 3.2 3.7h3.4v5.3h-6.6c-4.4 0-6.7-2.4-6.7-7.1V7.7zM114.7 11.8h6.7v26.6h-6.7zM114.7 2.2h6.7v5.5h-6.7z"
      />
      <path
        class="prefix__st0"
        d="M145.4 38.4c-5.9 9.3-16.2 15.4-28 15.4s-22.1-6.1-28-15.4c6.9 6.4 16.9 10.3 28 10.3 11.1.1 21.1-3.9 28-10.3"
      />
      <path
        d="M.8 32.3v-3.6c1.9.8 4.2 1.2 6.7 1.2 1.9 0 2.8-.6 2.8-1.8 0-1.1-.6-1.7-1.9-1.7H5.3c-3.5 0-5.3-1.8-5.3-5.3 0-3.6 2.6-5.4 7.7-5.4 2.2 0 4.2.3 6.2 1v3.6c-1.9-.8-4-1.2-6.3-1.2-2.4 0-3.5.6-3.5 1.8 0 1.1.7 1.7 2.1 1.7H9c3.8 0 5.8 1.8 5.8 5.3 0 3.6-2.5 5.4-7.4 5.4-2.5 0-4.6-.4-6.6-1m16.9.8V15.8h3.4l.5 2.2c2-1.5 4-2.2 5.9-2.2 2.2 0 3.7.8 4.6 2.4 2.2-1.6 4.3-2.4 6.2-2.4 3.7 0 5.5 2.2 5.5 6.6v10.7h-4.4V22.2c0-1.9-.8-2.9-2.4-2.9-1.3 0-2.7.7-4.1 2.2v11.6h-4.4V22.2c0-2-.8-2.9-2.4-2.9-1.4 0-2.8.7-4.1 2.2v11.6h-4.3zm29.4-5.4c0-3.6 2.5-5.3 7.5-5.3 1.5 0 2.9.1 4 .3v-1.1c0-1.5-1.5-2.3-4.5-2.3-1.9 0-3.8.3-5.6.8v-3.5c1.9-.5 3.8-.8 5.6-.8 5.9 0 8.9 1.9 8.9 5.7v11.6h-2.2l-1.9-1.2c-1.5.8-3.1 1.2-4.8 1.2-4.7 0-7-1.8-7-5.4zM58.6 26c-1.1-.2-2.4-.3-4-.3-2.1 0-3.2.7-3.2 2s.9 2 2.6 2c1.8 0 3.3-.5 4.5-1.4V26zm8.7 7.1V15.8h3.4l.5 2.2c1.5-1.5 3.1-2.2 4.8-2.2v3.5c-1.6 0-3.1.7-4.4 2.2v11.6h-4.3zM78.7 13h3.2l.7 2.8h4v3.5h-3.5v8c0 1.6.7 2.4 2.1 2.4h1.4v3.5H83c-2.9 0-4.4-1.6-4.4-4.7V13z"
        fill="#fff"
      />
    </svg>
    <p
      class="c34"
      color="#fff"
      data-testid="text_Text"
    >
      Todos os direitos reservados - 2020
    </p>
  </div>
</div>
`;
